src/promptflow/promptflow/batch/_batch_engine.py

```diff
@@ -210,6 +221,66 @@ def run(
                 )
                 raise unexpected_error from e
 
+    def _copy_previous_run_result(
+        self,
+        resume_from_run_storage: AbstractBatchRunStorage,
+        resume_from_run_output_dir: Path,
+        batch_inputs: List,
+    ) -> List[LineResult]:
```

Comment:
Please copy everything other than "output.jsonl" from resume_from_run_output_dir to the current output dir.
They should be the referenced image which need to exist in the current output dir so that the copied lines in the new output.jsonl would be valid.